github/git_blobs_test.go

```diff
@@ -51,6 +52,28 @@ func TestGitService_GetBlob_invalidOwner(t *testing.T) {
 	testURLParseError(t, err)
 }
 
+func TestGitService_GetBlobRaw(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/git/blobs/s", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", "application/vnd.github.v3.raw")
+
+		fmt.Fprint(w, `raw contents here`)
+	})
+
+	blob, _, err := client.Git.GetBlobRaw(context.Background(), "o", "r", "s")
+	if err != nil {
+		t.Errorf("Git.GetBlob returned error: %v", err)
```

Comment:
Minor, should be `GetBlobRaw` in the error text.

Reply:
whoops. fixed.